data-raw/colour_aliases_1in1000.R

```diff
@@ -0,0 +1,83 @@
+colours_1in1000 <- list(
```

Comment:
Maybe create the full dataset from its normalized pieces? The result is the same but the code might be a lil easier to read and maintain.

``` r
name_alias <- tibble::tribble(
  ~name,                 ~alias,
  "red",      "late transition",
  "red",          "late action",
  "red", "high carbon exposure",
)

name_hex <- tibble::tribble(
  ~name,       ~hex,
  "red",  "#F53D3F",
)

dplyr::left_join(name_alias, name_hex)
#> Joining, by = "name"
#> # A tibble: 3 × 3
#>   name  alias                hex    
#>   <chr> <chr>                <chr>  
#> 1 red   late transition      #F53D3F
#> 2 red   late action          #F53D3F
#> 3 red   high carbon exposure #F53D3F
```

<sup>Created on 2022-01-24 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

Reply:
so far as I know ggplot likes named vectors for color names, but doesn't really know how to deal with dataframes of colors. from [the docs](https://ggplot2.tidyverse.org/reference/scale_manual.html)
> If this is a named vector, then the values will be matched based on the names instead.

Reply:
@maurolepore indeed as Alex indicated - the named vectors are the way to go here if we want it to work with `ggplot2` which I personally find awesome.